domains/fengkch.json

```diff
@@ -0,0 +1,12 @@
+
+    {
+      "owner": {
+        "username": "sohokay",
+        "email": "kaichangfeng@gmail.com"
+      },
+
+      "record": {
+        "A": ["https://github.com/sohokay"]
```

Comment:
Please change this to a `URL` record and not use an array.

Like this: `"URL": "https://github.com/sohokay"`